bin/package.js

```diff
@@ -410,32 +416,82 @@ function buildWin32 (cb) {
         noMsi: true,
         outputDirectory: DIST_PATH,
         productName: config.APP_NAME,
-        remoteReleases: config.GITHUB_URL,
-        setupExe: config.APP_NAME + 'Setup-v' + config.APP_VERSION + '.exe',
+        /**
+         * Only create delta updates for the Windows x64 build because 90% of our
+         * users have Windows x64 and the delta files take a *very* long time to
+         * generate. Also, the ia32 files on GitHub have non-standard Squirrel
+         * names (i.e. RELEASES-ia32 instead of RELEASES) and so Squirrel won't
+         * find them unless we proxy the requests.
+         */
+        remoteReleases: destArch === 'x64'
+          ? config.GITHUB_URL
+          : undefined,
+        /**
+         * If you hit a "GitHub API rate limit exceeded" error, set this token!
+         */
+        remoteToken: process.env.WEBTORRENT_GITHUB_API_TOKEN,
+        setupExe: config.APP_NAME + 'Setup-v' + config.APP_VERSION + archStr + '.exe',
         setupIcon: config.APP_ICON + '.ico',
         signWithParams: signWithParams,
         title: config.APP_NAME,
         usePackageJson: false,
         version: pkg.version
       })
       .then(function () {
-        console.log('Windows: Created installer.')
+        console.log(`Windows: Created ${destArch} installer.`)
+
+        /**
+         * Delete extraneous Squirrel files (i.e. *.nupkg delta files for older
+         * versions of the app)
+         */
+        fs.readdirSync(DIST_PATH)
+          .filter((name) => name.endsWith('.nupkg') && !name.includes(pkg.version))
+          .forEach((filename) => {
+            fs.unlinkSync(path.join(DIST_PATH, filename))
+          })
+
+        if (destArch === 'ia32') {
+          console.log('Windows: Renaming ia32 installer files...')
+
+          // RELEASES -> RELEASES-ia32
+          var relPath = path.join(DIST_PATH, 'RELEASES-ia32')
+          fs.renameSync(
+            path.join(DIST_PATH, 'RELEASES'),
+            relPath
+          )
+
+          // WebTorrent-vX.X.X-full.nupkg -> WebTorrent-vX.X.X-ia32-full.nupkg
+          fs.renameSync(
+            path.join(DIST_PATH, `${config.BUILD_NAME}-full.nupkg`),
+            path.join(DIST_PATH, `${config.BUILD_NAME}-ia32-full.nupkg`)
+          )
+
+          // Change file name inside RELEASES-ia32 to match renamed file
+          var relContent = fs.readFileSync(relPath, 'utf8')
+          relContent = relContent.replace(/full\.nupkg$/, '-ia32-full.nupkg')
```

Comment:
Might want to do something like this, just in case:
```js
var relContent32 = relContent.replace(...)
if (!relContent32.length || relContent32 === relContent) {
  throw new Error(...)
}
```